packages/ui/src/components/selectors/GenericAccountSelection.tsx

```diff
@@ -119,6 +123,41 @@ const GenericAccountSelection = ({
     [onInputBlur]
   )
 
+  const getRenderOption = (props: any, option: any) => {
```

Comment:
Can we avoid this `any` please?

Reply:
PS. add linter rule to prevent it 😛

Reply:
@mpetrunic for sure, but in the separate PR 😛